includes/class-logger.php

```diff
@@ -68,6 +68,10 @@ public static function init_logger() {
 	 * @return bool Depending on the enable_logging setting.
 	 */
 	public static function can_log() {
+		if ( apply_filters( 'wcpay_force_logging', false ) ) {
```

Comment:
It was done in order to allow the logger to actually log. Otherwise, during testing, the logging environment is not set up and it's impossible to test whether something was logged.

This could have been done earlier, but it seems like none of the other tests necessitated it.